introduction_to_amazon_algorithms/object_detection_tensorflow/Amazon_Tensorflow_Object_Detection.ipynb

```diff
@@ -0,0 +1,973 @@
+{
```

Comment:
<p>nit: sub-directoriey -&gt; sub-directory</p><p> </p><p>Also, is pennfudanped dataset approved by legal?</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3624/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
corrected.